Sources/ArcGISToolkit/Components/Offline/OfflineMapAreaMetadataView.swift

```diff
@@ -39,26 +39,20 @@ struct OfflineMapAreaMetadataView<Metadata: OfflineMapAreaMetadata>: View {
                 }
```

Comment:
```suggestion
                }
                .textCase(nil)
```
Suggested to remove upper case section title, as per the design.

Reply:
I'll check with Peggy, but she probably doesn't know that uppercase is the default for section headers. I'd rather not mess with defaults.